generators/import-jdl/index.js

```diff
@@ -33,7 +33,8 @@ function importJDL() {
   const jdlImporter = jhiCore.JDLImporter.createImporterFromFiles(this.jdlFiles, {
     databaseType: this.prodDatabaseType,
     applicationType: this.applicationType,
-    applicationName: this.baseName
+    applicationName: this.baseName,
+    forceNoFiltering: true
```

Comment:
Is this change still needed? I don't think it is.

Reply:
I think there isn't a down side, and prevent problems with upgrading:
- If you upgrade generator-jhipster-ionic from 5.1.0 to 5.1.1
The files will not be recreated.

Your call.